### ISSUE TITLE
false-positive errors with pyroute2 0.7.6

### ISSUE THREAD

**Issue by gthiemonge** [PyCQA/pylint#8497]
### Bug description

When using pylint with pyroute2 0.7.6, it detects false-positive issues:

The following code is valid pyroute2 code:

```python
import sys

import pyroute2

with pyroute2.IPRoute() as ipr:
    idx = ipr.link_lookup(ifname=sys.argv[1])[0]
    link = ipr.get_links(idx)[0]
    print(link)
```

But with the pyroute2 0.7.6 release, pylint detects some false-positive problems:

```
$ pip install pyroute2 pylint
Collecting pyroute2
  Using cached pyroute2-0.7.6-py3-none-any.whl (454 kB)
Collecting pylint
  Using cached pylint-2.17.1-py3-none-any.whl (535 kB)
[...]
$ pylint test_pyroute.py
************* Module test_pyroute
test_pyroute.py:7:11: E1136: Value 'ipr.get_links(idx)' is unsubscriptable (unsubscriptable-object)
test_pyroute.py:7:11: E1121: Too many positional arguments for method call (too-many-function-args)
```

https://github.com/svinota/pyroute2/blob/master/pyroute2/iproute/linux.py#L434

the get_links function takes positional arguments and returns a list.

(Note that pylint works fine with pyroute2 0.7.5)

### Command used

```shell
pylint test_pyroute.py
```

### Pylint output

```shell
************* Module test_pyroute
test_pyroute.py:7:11: E1136: Value 'ipr.get_links(idx)' is unsubscriptable (unsubscriptable-object)
test_pyroute.py:7:11: E1121: Too many positional arguments for method call (too-many-function-args)
```

### Expected behavior

No error

### Pylint version

```shell
pylint 2.17.1
astroid 2.15.1
Python 3.11.1 (main, Jan  6 2023, 00:00:00) [GCC 12.2.1 20221121 (Red Hat 12.2.1-4)]
```

### OS / Environment

Fedora 37

### Additional dependencies

pyroute2==0.7.6
